spec/requests/v0/caregivers_assistance_claims_spec.rb

```diff
@@ -189,75 +189,158 @@
       FileUtils.rm_f(response_pdf)
     end
 
-    it 'returns a completed PDF', run_at: '2017-07-25 00:00:00 -0400' do
-      expect(SavedClaim::CaregiversAssistanceClaim).to receive(:new).with(
-        form: form_data
-      ).and_return(
-        claim
-      )
+    context 'caregiver1010 toggle off' do
```

Comment:
The two context for on/off have the exact same assertions, since we expect them to be the same. The diff is larger since I broke them into two contexts.